src/tests/test_mem_btree.cpp

```diff
@@ -439,32 +439,6 @@ TYPED_TEST(BtreeTest, RangeUpdate) {
     this->query_validate(0, num_entries - 1, 75);
 }
 
-TYPED_TEST(BtreeTest, SimpleRemoveRange) {
```

Comment:
a very primitive UT (not needed)

Reply:
I would suggest not to remove this UT. I made changes in the btree logic and this test is the only one which could find me the bug, rest passed, so we don't need to throw away this tests.